src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.props

```diff
@@ -37,6 +37,7 @@
 
   <!-- Common BuildTools properties -->
   <PropertyGroup>
+    <RootIntermediateOutputPath Condition="$(RootIntermediateOutputPath) == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), init-tools.msbuild))/bin/obj/</RootIntermediateOutputPath>
```

Comment:
I don't think we want to rely on `init-tools.msbuild` being around.  Can we just assume this file is 1 level below the "RepoRoot"?  And if that doesn't work for folks, they can just set RootIntermediateOutputPath themselves.

Reply:
right but with what I had before I had that really ugly path: "C:\git\corefx\Tools\..\bin/....". I wanted to avoid that and currently all projects to have a init-tools.msbuild